Sources/KlaviyoSwift/StateManagement.swift

```diff
@@ -481,6 +486,18 @@ struct KlaviyoReducer: ReducerProtocol {
             pendingProfile[key] = value
             state.pendingProfile = pendingProfile
             return .none
+
+        case let .resetStateAndDequeue(request, invalidFields):
```

Comment:
I might have modified the deque action like this instead to include invalidFields. You are effectively chaining two actions together when you could update the state in one go.

Reply:
Yeah I initially had it that way but decided to create a separate action for this mainly because there are other code paths that call into the dequeue action and I didn't want to add code that is specific to a very small edge case in there.